app/views/edit/build-config.html

```diff
@@ -149,14 +149,14 @@
                                 <label>Image Source From</label>
                                 <ui-select required ng-model="imageOptions.fromSource.type" search-enabled="false">
                                   <ui-select-match>{{$select.selected | startCase}}</ui-select-match>
-                                  <ui-select-choices repeat="type in (imageSourceTypes | filter : $select.search)">
+                                  <ui-select-choices repeat="type in imageSourceTypes">
                                     <div ng-bind-html="type | startCase"></div>
```

Comment:
If you're not highlighting a search match, you shouldn't need to use `ng-bind-html`. Does this work?

```html
<ui-select-choices repeat="type in imageSourceTypes">
  {{type | startCase}}
</ui-select-choices>
```

Here and the selects below.